app/pages/system/networking/IpPoolPage.tsx

```diff
@@ -70,16 +73,54 @@ IpPoolPage.loader = async function ({ params }: LoaderFunctionArgs) {
 export function IpPoolPage() {
   const poolSelector = useIpPoolSelector()
   const { data: pool } = usePrefetchedApiQuery('ipPoolView', { path: poolSelector })
+  const { data: ranges } = usePrefetchedApiQuery('ipPoolRangeList', {
+    path: poolSelector,
+    query,
+  })
+  const navigate = useNavigate()
+  const deletePool = useApiMutation('ipPoolDelete', {
```

Comment:
See https://github.com/oxidecomputer/console/issues/2367. Do

```ts
const { mutateAsync: deletePool } = useApiMutation(...)
```

and change `deletePool.mutateAsync()` to `deletePool()` accordingly. Unfortunately, simply changing `deletePool` to `deletePool.mutateAsync` in the dep array doesn't satisfy the linter.